apps/export/factories.py

```diff
@@ -0,0 +1,17 @@
+import factory
+from factory.django import DjangoModelFactory
+
+from project.factories import ProjectFactory
+from user.factories import UserFactory
+from .models import Export
+
+
+class ExportFactory(DjangoModelFactory):
+    title = factory.Sequence(lambda n: f'Export-{n}')
+    exported_by = factory.SubFactory(UserFactory)
+    project = factory.SubFactory(ProjectFactory)
```

Comment:
Let's define this through test cases. Might be confusing when creating and trying to retrieve through a query. What do you think?